src/frames.py

```diff
@@ -64,7 +65,8 @@ def _gallery() -> None:
     def _shoot(self) -> None:
         """Take an image"""
         logger._log_info("Camera was clicked")
-        self.webcam.take_picture_and_save()
+        img_name = f"gallery/Image-{datetime.now().strftime('%Y-%m-%d-%H-%M-%S')}.jpg"
```

Comment:
@Trisanu-007 use same format for video files as well.

Reply:
Sure its somewhere in the fileIO file .